.github/workflows/release.yaml

```diff
@@ -0,0 +1,146 @@
+          run: sudo apt-get install -y mandoc scdoc ninja-build
+        - name: make dir structure
+          run: mkdir -p litani-${{ env.LITANI_VERSION }}/{DEBIAN,usr/{bin,libexec/litani,share/{doc/{litani},man/{man1,man5,man7}}}}
+        - name: Create a control file
+          run: |
+            touch litani-${{ env.LITANI_VERSION }}/DEBIAN/control
+            cat << EOF > litani-${{ env.LITANI_VERSION }}/DEBIAN/control
+            Package: Litani
+            Version: ${{ env.LITANI_VERSION }}
+            Architecture: amd64
+            Depends: ninja-build, gnuplot, graphviz, python3.9, python3-jinja2, python3-voluptuous
+            Maintainer: Kareem Khazem <karkhaz@amazon.co.uk>
+            Description: A program that provides platform-independent job control.
+              AWS Build Accumulator collects build jobs from multiple sources before executing them concurrently. It provides platform-independent job control (timeouts, return code control) and an output format that is easy to render into reports (for example, using the built-in renderer). AWS Build Accumulator shines where your project uses multiple different build systems or requires a unified interface describing heterogeneous build jobs.
+            EOF
+        - name: Clone Litani
+          run: sudo git clone https://github.com/awslabs/aws-build-accumulator.git /usr/libexec/litani
+        - name: Create Symlink
+          run: sudo ln -s /usr/libexec/litani/litani litani-${{ env.LITANI_VERSION }}/usr/bin/
+        - name: Copy to usr/libexec
+          run: |
+            cp -r /usr/libexec/litani/litani litani-${{ env.LITANI_VERSION }}/usr/libexec/litani/
+            cp -r /usr/libexec/litani/lib litani-${{ env.LITANI_VERSION }}/usr/libexec/litani/
+            cp -r /usr/libexec/litani/bin litani-${{ env.LITANI_VERSION }}/usr/libexec/litani/
+            cp -r /usr/libexec/litani/templates litani-${{ env.LITANI_VERSION }}/usr/libexec/litani/
+            cp -r /usr/libexec/litani/doc litani-${{ env.LITANI_VERSION }}/usr/libexec/litani/
+        - name: Move man pages and docs to usr/share
+          run: |
+            ./litani-${{ env.LITANI_VERSION }}/usr/libexec/litani/doc/configure && ninja
+            mv litani-${{ env.LITANI_VERSION }}/usr/libexec/litani/doc/out/man/*.1 litani-${{ env.LITANI_VERSION }}/usr/share/man/man1
+            mv litani-${{ env.LITANI_VERSION }}/usr/libexec/litani/doc/out/man/*.5 litani-${{ env.LITANI_VERSION }}/usr/share/man/man5
+            mv litani-${{ env.LITANI_VERSION }}/usr/libexec/litani/doc/out/man/*.7 litani-${{ env.LITANI_VERSION }}/usr/share/man/man7
+        - name: Remove unnecessary files
+          run: |
+            rm -rf litani-${{ env.LITANI_VERSION }}/usr/libexec/litani/doc
+            sudo rm -rf /usr/libexec/litani
+        - name: Create .deb package
+          id: create_packages
+          run: |
+            sudo dpkg-deb --build --root-owner-group litani-${{ env.LITANI_VERSION }}
```

Comment:
are `sudo` and `--root-owner-group` necessary here?

Reply:
`sudo` is required here.
And the `--root-owner-group` flag makes all deb package content owned by the root user, which is the standard way to go. Without such flag, all files and folders would be owned by the user, which might not exist in the system the deb package would be installed to.